addons/website_blog/views/website_blog_views.xml

```diff
@@ -77,6 +77,7 @@
                         <group name="publishing_details" string="Publishing Options">
                             <field name="author_id"/>
                             <field name="create_date" groups="base.group_no_one"/>
+                            <field name="visits" readonly="1"/>
```

Comment:
Move the readonly on the model (and remove it from the list view also while at it)